composer.json

```diff
@@ -29,7 +32,7 @@
         "composer-plugin-api": "^1.0 || ^2.0"
     },
     "require-dev": {
-        "composer/composer": "^2.1.9",
+        "composer/composer": "^2.1.9 || ^2.3.0-RC2",
```

Comment:
Uhm, `2.3.0-RC2` is already part of the `^2.1.9` selector :thinking:

Reply:
Perhaps you want `minimum-stability: rc`?

Reply:
yah, just wanted to allow RC versions but would prefer not doing so for *all* dependencies.
When changing `minimum-stability`, this will apply for all downstream dependencies - do we really want this?

Reply:
Changed to explicitly allow RC for `composer/composer`